articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,196 @@
+### Introduction
+SQL injection in [WordPress](https://kinsta.com/knowledgebase/what-is-wordpress/) is [ranked the second most critical vunerability](secure.wphackedhelp.com). [Updating your WordPress website](https://wordpress.org/support/article/updating-wordpress/) to the latest release shields you from SQL injection. Nevertheless, third party themes and plugins still pose a big threat to WordPress users. When you hear SQL injection, you understand it is the manipulation of a security flaw in database systems. WordPress as a platform runs server-side scripts in PHP. SQl and PHP hence make WordPress vulnerable to attackers via inserting malicious URLs embedded with commands that trigger behaviours from the database. Consequently,  sensitive information from your database is exposed or leaked to hackers which they can use to manipulate your WordPress website.
+
+#### What is SQLi?
+
+SQLi is an abbreviation for SQL injection.
+
```

Comment:
All instances of SQL should have a capital L. Please fix this where this occurs.